src/main/resources/com/coravy/hudson/plugins/github/GithubProjectProperty/help-statusContext.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
```

Comment:
And then just say in help that display name will be used for any UI places where job name is expected.